src/subdomains/core/buy-crypto/process/services/buy-crypto-registration.service.ts

```diff
@@ -77,16 +76,6 @@ export class BuyCryptoRegistrationService {
           if (result === ValidationError.PAY_IN_TOO_SMALL) {
             await this.payInService.ignorePayIn(payIn, PayInPurpose.BUY_CRYPTO, cryptoRoute);
             continue;
-          } else if (result === ValidationError.PAY_IN_NOT_SELLABLE) {
-            if (cryptoRoute.asset.blockchain === payIn.address.blockchain) {
-              await this.payInService.returnPayIn(
-                payIn,
-                PayInPurpose.BUY_CRYPTO,
-                BlockchainAddress.create(cryptoRoute.user.address, payIn.address.blockchain),
-                cryptoRoute,
-              );
-              continue;
-            }
```

Comment:
Question is do we need a enum here? Atm it would only be 'PayInTooSmall' for the enum. If thats the only one in future too we could change from enum to boolean or something 🤔

Reply:
Yes, we can probably change it